schort.py

```diff
@@ -5,23 +5,15 @@
 
 app = Flask(__name__)
 
-@app.route('/', methods=['GET', 'POST'])
-@app.route('/<shortLink>', methods=['GET', 'POST'])
+@app.route('/', methods=['GET', 'POST', 'HEAD'])
+@app.route('/<shortLink>', methods=['GET', 'POST', 'HEAD'])
```

Comment:
HEAD is actually implicit in flask as soon as you enable GET requests. Therefore not needed here. I'd prefer a short comment explaining that `# HEAD is implicit with flask`